mysqld_exporter.go

```diff
@@ -225,36 +241,42 @@ func (e *Exporter) setGenericMetric(name string, value float64) {
 	e.metrics[name].Set(value)
 }
 
-func (e *Exporter) setMetrics(scrapes <-chan []string) {
-	var comRegex = regexp.MustCompile(`^com_(.*)$`)
-	var connectionErrorRegex = regexp.MustCompile(`^connection_errors_(.*)$`)
-	var innodbRowRegex = regexp.MustCompile(`^innodb_rows_(.*)$`)
-	var performanceSchemaRegex = regexp.MustCompile(`^performance_schema_(.*)$`)
+var (
+	globalStatusRegexp = regexp.MustCompile(`^(com|connection_errors|innodb_rows|performance_schema)_(.*)$`)
```

Comment:
Since this is a single var now, you should just write

    var globalStatusRegexp = regexp...